_data/banking.yml

```diff
@@ -537,4 +537,4 @@ websites:
       url: http://zenbanx.ca/
       twitter: ZenBanx
       img: zenbanx.png
-      tfa: No
+      tfa: No
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.